.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,3 +1,18 @@
+<!-- 
```

Comment:
I like this concept a lot but I'd rather it not be in a comment -- comments make it harder to see what's going on in integrations like the Slack one because they hide the start of the content. I think they also make opening a PR seem a lot harder.

How about putting these recommendations in the `CONTRIBUTION_GUIDE.md` instead?

Reply:
Agreed to move those naming rules into `CONTRIBUTING.md`. 
As for comments, what about this:
```markdown
<!-- 
Thank you for contributing to ODK Collect !

Before sending this PR, I hope you have read the CONTRIBUTING.md first.
https://github.com/opendatakit/collect/blob/master/CONTRIBUTING.md
-->
```